src/lib.rs

```diff
@@ -0,0 +1,56 @@
+// `error_chain!` can recurse deeply
+#![recursion_limit = "1024"]
+
+#[macro_use]
+extern crate error_chain;
+
+#[macro_use]
+extern crate lazy_static;
+
+extern crate ansi_term;
+extern crate atty;
+extern crate console;
+extern crate content_inspector;
+extern crate dirs as dirs_rs;
+extern crate encoding;
+extern crate git2;
+extern crate shell_words;
+extern crate syntect;
+extern crate wild;
+
+pub mod assets;
+pub mod config;
```

Comment:
It feels to me like the command-line argument and environment variable handling should be part of the CLI, not the library.

Reply:
done 535ac2189889651cfc1311614898b686a0caa157